types/knex/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Knex.js 0.14
+// Type definitions for Knex.js 0.15.2
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
@DanielRosenwasser I am not sure I see where to do that. Could you point me at an example or what I need to tweak?

Reply:
Just remove the rules field entirely since they override the fields that they extend from `dtslint/dt.json`

Reply:
👍 After enabling the rules, there are dozens of failures. I've pushed up the change anyway.